checks.d/mysql.py

```diff
@@ -1188,7 +1188,7 @@ def _query_exec_time_per_schema(self, db):
         # Fetches the avg query execution time per schema and returns the
         # value in microseconds
 
-        sql_avg_query_run_time = """SELECT schema_name, SUM(count_star) cnt, ROUND(AVG(avg_timer_wait)/1000000) AS avg_us
+        sql_avg_query_run_time = """SELECT schema_name, SUM(count_star) cnt, ROUND((SUM(sum_timer_wait) / SUM(count_star)) / 1000000) AS avg_us
```

Comment:
kind of unrelated, but we could remove `SUM(count_star) cnt` from the `SELECT` as we don't use it right?